actionpack/test/controller/mime/respond_to_test.rb

```diff
@@ -549,6 +556,12 @@ def test_handle_any
     assert_equal "Either JS or XML", @response.body
   end
 
+  def test_handle_any_doesnt_set_request_content_type
```

Comment:
It's the response content type that's being set from the request Accept header. And the response is set, so it's a bit of a weird phrasing.